src/engine/Map.hxx

```diff
@@ -208,12 +200,12 @@ private:
   unsigned char getElevatedNeighborBitmask(Point centerCoordinates);
 
   /** \brief Change map node height.
-  * @param mapNode Map node to change height.
-  * @param higher if set to true make node higher, otherwise lower.
+  * @param coordinate of the map node to change height.
+  * @param elevate if set to true make node higher, otherwise lower.
   * @param neighbors All neighbor map nodes.
```

Comment:
Looks like you removed this parameter in this PR but forgot to remove the corresponding doxy line.